transformFiringRatesToLikelihoodMeasure.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+    Description
```

Comment:
You should put this docstring after the function so people can see it when they do help(transformFiringRatesToLikelihoodMeasure)

Reply:
and by after i mean
```
def myfunction():
"""docstring goes here


"""
    print('doing my function')
```

Reply:
Sorry, merge now, fix later.